src/tools/fuzzing/fuzzing.cpp

```diff
@@ -1212,6 +1216,14 @@ Expression* TranslateToFuzzReader::_makeunreachable() {
 }
 
 Expression* TranslateToFuzzReader::makeTrivial(Type type) {
+  struct TrivialNester {
```

Comment:
This is an unfortunate amount of boilerplate just to run `trivialNesting--` to run at scope exit, but I couldn't find a better way to do it outside of boost.

Reply:
Yeah... other languages can do better, e.g. `defer` in Go and Zig.